test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout.lua

```diff
@@ -0,0 +1,884 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.checkAllValidations = true
+
+--[[ Required Shared libraries ]]
+local json = require("modules/json")
+local utils = require('user_modules/utils')
+local actions = require("user_modules/sequences/actions")
+local commonRC = require('test_scripts/RC/commonRC')
+local runner = require('user_modules/script_runner')
+
+--[[ Tests Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Common Variables ]]
+local c = {}
+c.start = actions.start
+c.registerAppWOPTU = actions.registerAppWOPTU
+c.activateApp = actions.activateApp
+c.postconditions = actions.postconditions
+c.getMobileSession = actions.mobile.getSession
+c.getHMIConnection = actions.hmi.getConnection
+c.getHMIAppId = actions.app.getHMIId
+c.defineRAMode = commonRC.defineRAMode
+c.rpcAllowed = commonRC.rpcAllowed
+c.getAppEventName = commonRC.getAppEventName
+c.getAppRequestParams = commonRC.getAppRequestParams
+c.getHMIEventName = commonRC.getHMIEventName
+c.getHMIRequestParams = commonRC.getHMIRequestParams
+c.getHMIResponseParams = commonRC.getHMIResponseParams
+c.getConfigAppParams = actions.getConfigAppParams
+
+c.notificationTime = 0
+c.jsonFileToTable = utils.jsonFileToTable
+c.tableToJsonFile = utils.tableToJsonFile
+c.cloneTable = utils.cloneTable
+c.Step = runner.Step
+c.Title = runner.Title
+
+c.modules = { "RADIO", "CLIMATE" }
+
+c.rpcs = {}
+
+c.rpcsArray = {
+  "SendLocation",
+  "Alert",
+  "SubtleAlert",
+  "PerformInteraction",
+  "Slider",
+  "Speak",
+  "ScrollableMessage",
+  "DiagnosticMessage",
+  "SetInteriorVehicleData",
+  "CreateInteractionChoiceSet",
+  "DeleteInteractionChoiceSet",
+  "DeleteSubMenu",
```

Comment:
Should "AddSubMenu" not also be included in the rpcsArray?

Reply:
@ShobhitAd Please find update in 557e4e0